FORGE/ANTONIO/myFunctions.py

```diff
@@ -1,31 +1,36 @@
+from cmath import log
+from math import e
+import math
+
+
 def myfunction():
     return "hi"
 
 def function2():
     return "ola"
 
-from cmath import log
-import numpy
 
 
 
-def FieldCapacity(C,D,S,OM,TP):
+def FieldCapacity(C,S,D,OM,TP):
```

Comment:
You swaped these two arguments... if you swap  the order, then the equation returns  0.6056921297711275 with  FieldCapacity(0.2,1.3,0.1,0.05,1)